README.md

```diff
@@ -108,30 +110,22 @@ MSVC builds of Rust additionally require an installation of Visual Studio 2013
 (or later) so `rustc` can use its linker. Make sure to check the “C++ tools”
 option.
 
-With these dependencies installed, the build takes two steps:
+With these dependencies installed, you can build the compiler in a `cmd.exe`
+shell with:
 
 ```sh
-$ ./configure
-$ make && make install
+$ python x.py build
```

Comment:
This prompt is supposed to be `cmd.exe` according to the above paragraph, so using `$` is incorrect here.